ocamlorg.opam

```diff
@@ -70,9 +70,6 @@ build: [
 ]
 dev-repo: "git+https://github.com/ocaml/ocaml.org.git"
 pin-depends: [
-  # See https://gitlab.com/gerdstolpmann/lib-ocamlnet3/-/merge_requests/24
-  # See https://github.com/aantron/ocamlnet/commit/7ef255a0a994ef2721f3359077b36bcdd0428ee4
-  ["ocamlnet.dev" "git+https://github.com/aantron/ocamlnet#7ef255a0a994ef2721f3359077b36bcdd0428ee4"]
```

Comment:
I haven't been able to get this generated from a `dune-workspace` `(pin ...)` stanza. Do you have any experience as to why this isn't being generated in this commit?

Reply:
I've updated the `.opam.template` file. This is no longer an issue.